.travis.yml

```diff
@@ -55,16 +55,42 @@ script:
       presto-product-tests/bin/run_on_docker.sh \
         singlenode-kerberos-hdfs-impersonation -g storage_formats,cli,hdfs_impersonation,authorization
   - |
+    # Build presto-server-rpm for later artifact upload
+    test ! [[ -v DEPLOY_S3_SECRET_KEY_ID && -v PRODUCT_TESTS ]] ||
+       ./mvnw install -DskipTests=true $MAVEN_SKIP_CHECKS_AND_DOCS -B -q -T C1 -pl 'presto-server-rpm'
```

Comment:
This can just be `-DskipTests`. Reference: http://maven.apache.org/surefire/maven-surefire-plugin/examples/skipping-test.html